cmake/dependencies/AMReX.cmake

```
@@ -82,7 +82,7 @@ option(pyAMReX_amrex_internal "Download & build AMReX" ON)
 set(pyAMReX_amrex_repo "https://github.com/AMReX-Codes/amrex.git"
     CACHE STRING
     "Repository URI to pull and build AMReX from if(pyAMReX_amrex_internal)")
-set(pyAMReX_amrex_branch "23.04"
+set(pyAMReX_amrex_branch "8cae8024915ff8d31fe655665d17783fd34cda8f"
```

https://github.com/AMReX-Codes/amrex/commit/bb47ec9a6e4fbaea36839d94cd2bf7bdba038168
```suggestion
set(pyAMReX_amrex_branch "bb47ec9a6e4fbaea36839d94cd2bf7bdba038168"
```